pkg/integrations/v2/app_agent_receiver/payload.go

```diff
@@ -76,10 +77,15 @@ func (e Exception) KeyVal() *KeyVal {
 	KeyValAdd(kv, "type", e.Type)
 	KeyValAdd(kv, "value", e.Value)
 	KeyValAdd(kv, "stacktrace", e.String())
+	MergeKeyValWithPrefix(kv, KeyValFromMap(e.Context), "context_")
```

Comment:
Q: so this means the map will be something like:
context_component: ReactErrorBoundary
context_ReactError: Annoying Error
? (just checking if this is how it works)

Reply:
Yes. I kept it the same concept as for the Log Context (see line 201)